### DISCUSSION TITLE
Record key codes and timing

### DISCUSSION THREAD

**Discussion by angoodkind** [jspsych/jsPsych#1517] [Q&A]
This may be too vague, but I am having a difficult time writing an experiment that records all of the key codes and timing for a free text entry experiment. I see some issues have addressed this, but am having trouble putting it all together.

**Comment by jodeleeuw** [comment]
Going to convert this over to a discussion. Not sure if you've spotted the discussions that are separate from issues, but this has been talked about there, too. Some of those threads do have some example code.

 I think as it currently stands it is too open-ended for us to provide solid advice with a reasonable time investment, but if there are specific issues that you run into we can probably help more.

**Reply by angoodkind** [reply]
Thanks Josh! I didn't know about Discussions. I am trying to modify the code in Discussion #772, as it looks like it's doing exactly what I aim for. Do you have any pointers on how to expand your code sample, and also where js/utilities.js (line 13 in the experiment) is located?
